gymnasium/envs/box2d/lunar_lander.py

```diff
@@ -403,6 +401,10 @@ def reset(
             True,
         )
 
+        if self.enable_wind:  # Initialize wind pattern based on index
+            self.wind_idx = np.random.randint(-9999, 9999)
```

Comment:
`self.np_random.` As Mark said, in ththie issue

Reply:
Yes, sorry - I did not understand before. It has been fixed now.

Reply:
I cannot get the all the doctests to run on my machine (also not on the main branch). Is there a command to only test the updated files?
![pytest](https://github.com/Farama-Foundation/Gymnasium/assets/22000796/b6937227-282a-4f29-9439-e181d875f128)

Reply:
Don't worry, this passes all the CI testing